scripts/stac/imagery/capture_aera.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Any, Dict, List
+
+from shapely import BufferCapStyle, BufferJoinStyle, Geometry, to_geojson, union_all
+from shapely.geometry import Polygon
+
+DECIMAL_DEGREES_1M = 0.00001
+"""
+Degree precision of ~1m (decimal places 5, https://en.wikipedia.org/wiki/Decimal_degrees)
+"""
+
+
+def to_feature(geometry: Geometry) -> Dict[str, Any]:
```

Comment:
Is there a GeoJSON `Feature` type we could use as the return type, or possibly a `TypedDict` to avoid the `Any`?

Reply:
I spent some time looking at this today:
- there is a `geojson` library that I've been using in a previous commit in this PR but it does round the coordinates and I could not find a way to avoid it
- Creating a TypedDict seems very complex (managing all kind of `coordinates` field that are different for each Geometry Type for example).
I think at the moment it will make the code over-complicated, so I pass on this.